config.py

```diff
@@ -0,0 +1,33 @@
+# This file contains most of the configuration variables that your app needs.
+import os
+from datetime import timedelta
+BASE_DIR = os.path.abspath(os.path.dirname(__file__))
+
+
+class Config(object):
+    REMEMBER_COOKIE_DURATION = timedelta(days=14)
+    SECRET_KEY = "\xf9'\xe4p(\xa9\x12\x1a!\x94\x8d\x1c\x99l\xc7\xb7e\xc7c\x86\x02MJ\xa0"
```

Comment:
You might want to hide private information such as your secret key in a `.env` file. Check this package: https://github.com/theskumar/python-dotenv
 it helps with storing private variables as environment variables.